projects/Mallard/src/helpers/push-notifications.ts

```diff
@@ -63,12 +65,20 @@ const pushNotifcationRegistration = () => {
                 })
             }
         },
-        onNotification: (notification: any) => {
+        onNotification: async (notification: any) => {
             const key =
                 Platform.OS === 'ios' ? notification.data.key : notification.key
             if (key) {
                 const screenSize = imageForScreenSize()
-                downloadAndUnzipIssue(key, screenSize)
+                const issueSummaries = await getIssueSummary().getValue()
+                // Check to see if we can find the image summary for the one that is pushed
+                const pushImageSummary = issueSummaries.find(
+                    o => o.localId === `${defaultSettings.appPrefix}/${key}`,
```

Comment:
We're creating this localId in the same format twice - maybe it can be a common function to build it?